src/multiple-choice/components/runtime.tsx

```diff
@@ -117,6 +134,16 @@ export const Runtime: React.FC<IProps> = ({ authoredState, interactiveState, set
             readOnly={readOnly}
             disabled={readOnly}
           />
+          {choice.correct !== undefined && choice.correct === true && showCorrect &&
+            <div className={css.markContainer}>
+              <CheckMarkIcon className={`${css.mark} ${css.correctCheck}`} />
+            </div>
+          }
+          {choice.correct !== undefined && choice.correct === false && showDistractor &&
```

Comment:
```suggestion
          {!choice.correct && showDistractor &&
```
As above, the test against `undefined` is redundant (if it's `false` it can't be `undefined`). The specific test for `false` may still be necessary if other falsy values (e.g. `undefined`) should _not_ be marked, but if that's not the case I would opt for the simpler test.

Reply:
Same as above.  Thanks!  Fixed.